client/forgotPassword/forgotPassword.ctrl.js

```diff
@@ -22,4 +22,20 @@ module.exports = class ForgotPassword {
       .then(() => this.$scope.$apply())
       .catch(this.errorHandler);
   }
+  sendUsernameRequest () {
+    return Promise.resolve(this.$http({
+      method: 'POST',
+      url: `/api/v1/user/${this.resetEmail}/forgotUsername`
```

Comment:
Not sure how much of a concern this is, but we should probably URL-encode the email address in case there are special characters for some reason.